auth-web/src/components/auth/common/AccountBusinessType.vue

```diff
@@ -232,11 +232,12 @@ export default defineComponent({
 
     const {
       businessSizeCodes,
-      businessTypeCodes
+      businessTypeCodes,
+      governmentTypeCodes
     } = storeToRefs(codesStore)
```

Comment:
Couldn't you do the same for currentOrganization and isCurrentGovnOrg?

Reply:
no, I did before, it was a bug. currentOrganization's attributes changing while creating account